### PR TITLE
CONTRIBUTING.md Review

### LINKED ISSUE #33: Need for "How to Contribute to Issues" Documentation
### Issue Name:

Need for "How to Contribute to Issues" Documentation

#### Context:

At the current stage, there is no documentation if an open-source user wants to contribute to the BitCamp (or any repository in general) repo. We need a markdown file explaining the steps of contributing to an open-source repository, specifically, one of ours. In this file, we need steps on finding issues that are open for open-source users (and different fields: newcomers, advanced, etc) and steps on contributing and actually handling the issue (forking the repo, creating a PR and notifying Bit Project members).

#### Type:

Issue Documentation Addition

#### Actual Issue:

Need for "How to Contribute" Documentation

### Desired Outcome:

A markdown file filled with steps through information and photos that explain, in a simple way, how an open-source user can contribute to a Bit Project repository. In this file, we need steps on finding issues that are open for open-source users (and different fields: newcomers, advanced, etc) and steps on contributing and actually handling the issue (forking the repo, creating a PR and notifying Bit Project members).

### (Optional) Hint/Example:

None.

### Possible Solutions:

There are many examples of other open-source repositories that already have "how to contribute" issues and sites that give good step to step tutorials (https://github.blog/2020-01-22-browse-good-first-issues-to-start-contributing-to-open-source/)

### Other Details:

None.

### PR DESCRIPTION
**Pull request by beaubhp**
@emsesc @lazyplatypus CONTRIBUTING.md isn't done but wanted to receive some feedback before I continue. This part is for newcomers to understand how we label our issues. Next, I'm going to create a "getting started" and "contribution" part!

### CODE REVIEW

**Review by lazyplatypus — CHANGES_REQUESTED** [review]
I think we need to have a basics to Github under a new markdown file and be like. here is our guide to getting started with github if you are new to it. CONTRIBUTING.md almost never includes a github guide. 

The type of issues, on the other is very relevant so keep! @emsesc @MinyMeep1 thoughts?

**Comment by beaubhp** [comment]
@lazyplatypus I agree with having another markdown file that is based on the basics of GitHub but I think we should have a few links/only a few helpful terms that are really important when contributing to issues.

I'm going to continue the markdown file with steps on how to actually contribute and end it with a section on notifying team members and creating a PR, any thoughts on the format?

**Comment by lazyplatypus** [comment]
I think it looks good! @MinyMeep1  might be more helpful here

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# How to Contribute (*Bit Project Style*)
+
```
Comment (emsesc) [inline CONTRIBUTING.md:2]: Consider adding a resource page with videos/github docs. Here's also a learning lab from github: https://lab.github.com/githubtraining/introduction-to-github

> I think we need to have a basics to Github under a new markdown file and be like. here is our guide to getting started with github if you are new to it. CONTRIBUTING.md almost never includes a github guide.

Referencing Daniel's comment, I agree that the resources + basic github stuff should be in another md file

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+Welcome open-source user! We're glad to have you here. You're probably interested in contributing to our open-source repositories here at Bit Project. Before we go over the standards of contributing to our repos, let's go over few key points to help you get started.
+
+First of all, how it works. While contributing might seem quick and easy, the process is quite difficult to the regular newcomer. If you have some knowledge of Github (example: what an issue is, forking, creating a pull request) then feel free to skip this part. If not, here is a quick summary of some important terms and a few helpful links if you're just starting with Github.
+
+---
+
+|  Term | Meaning  |
+| ------------ | ------------ |
+| Issue  | Issues are a great way to keep track of tasks, enhancements, and bugs for your projects. They’re kind of like email—except they can be shared and discussed with the rest of your team. Most software projects have a bug tracker of some kind. GitHub’s tracker is called Issues, and has its own section in every repository.  |
```
Comment (emsesc) [inline CONTRIBUTING.md:13]: link github docs with the corresponding terms

**Review by emsesc — COMMENTED** [review]
Overall, great explainations and format. 
> I'm going to continue the markdown file with steps on how to actually contribute and end it with a section on notifying team members and creating a PR, any thoughts on the format?

^^ This looks good. This is already included in the issues template, but maybe also clarify here which issues are members only.

File: CONTRIBUTING.md (hunk 1 above)
Comment (beaubhp) [reply CONTRIBUTING.md:2]: Yup! Updated with links, good suggestion.

**Comment by emsesc** [comment]
@Bahburs Would recommend adding that a PR should immediately be made and marked as a draft. Helps to get feedback and see progress

**Comment by emsesc** [comment]
added a "checklist" that contains items that contributors should consider reviewing. @lazyplatypus

**Comment by emsesc** [comment]
Thoughts on new additions/reformatting @lazyplatypus @Bahburs ?

**Review by lazyplatypus — APPROVED** [review]

**Review by beaubhp — COMMENTED** [review]
Looks good. I think down the line I'm going to make it look a bit more beginner friendly by adding some helpful images/GIFS, is that fine @lazyplatypus?

**Pull request merged**